tests/core/test_search.py

```diff
@@ -256,51 +258,170 @@ def test_create_vespa_search_params(
-    "filters",
+    "filters, expected",
     [
-        None,
-        {FilterField.CATEGORY: ["Executive"]},
-        {FilterField.LANGUAGE: ["english"]},
-        {FilterField.SOURCE: ["CCLW"]},
-        {FilterField.REGION: ["europe-central-asia"]},
-        {FilterField.COUNTRY: ["france", "germany"]},
-        {FilterField.COUNTRY: ["cambodia"]},
-        {FilterField.REGION: ["south_america"], FilterField.COUNTRY: ["france"]},
+        (None, None),
+        ({FilterField.REGION: ["this-is-not-a-region"]}, None),
+        (
+            {
+                FilterField.REGION: ["latin-america-caribbean"],
+                FilterField.COUNTRY: ["france"],
+            },
+            None,
+        ),
+        ({FilterField.REGION: ["north-america"]}, {"family_geography": ["CAN", "USA"]}),
+        (
+            {
+                FilterField.REGION: ["north-america"],
+                FilterField.COUNTRY: ["not-a-country"],
+            },
+            {"family_geography": ["CAN", "USA"]},
+        ),
+        (
+            {FilterField.REGION: ["north-america"], FilterField.COUNTRY: ["canada"]},
+            {"family_geography": ["CAN"]},
+        ),
+        ({FilterField.COUNTRY: ["cambodia"]}, {"family_geography": ["KHM"]}),
+        ({FilterField.COUNTRY: ["this-is-not-valid"]}, None),
+        (
+            {FilterField.COUNTRY: ["france", "germany"]},
+            {"family_geography": ["FRA", "DEU"]},
+        ),
+        (
+            {FilterField.COUNTRY: ["cambodia"], FilterField.CATEGORY: ["Executive"]},
+            {"family_category": ["Executive"], "family_geography": ["KHM"]},
+        ),
+        (
+            {FilterField.COUNTRY: ["cambodia"], FilterField.LANGUAGE: ["english"]},
+            {"document_languages": ["english"], "family_geography": ["KHM"]},
+        ),
+        (
+            {FilterField.COUNTRY: ["cambodia"], FilterField.SOURCE: ["CCLW"]},
+            {"family_source": ["CCLW"], "family_geography": ["KHM"]},
+        ),
+        (
+            {
+                FilterField.REGION: ["north-america"],
+                FilterField.CATEGORY: ["Executive"],
+            },
+            {"family_category": ["Executive"], "family_geography": ["CAN", "USA"]},
+        ),
+        (
+            {FilterField.REGION: ["north-america"], FilterField.LANGUAGE: ["english"]},
+            {"document_languages": ["english"], "family_geography": ["CAN", "USA"]},
+        ),
+        (
+            {FilterField.REGION: ["north-america"], FilterField.SOURCE: ["CCLW"]},
+            {"family_source": ["CCLW"], "family_geography": ["CAN", "USA"]},
+        ),
+        ({FilterField.CATEGORY: ["Executive"]}, {"family_category": ["Executive"]}),
+        ({FilterField.LANGUAGE: ["english"]}, {"document_languages": ["english"]}),
+        ({FilterField.SOURCE: ["CCLW"]}, {"family_source": ["CCLW"]}),
     ],
 )
-def test__convert_filters(test_db, filters):
+def test__convert_filters(test_db, filters, expected):
```

Comment:
Is that a typo __

Reply:
Nope, the function is `_convert_filters`